.chromatic-fc/custom-addons/chromatic/index.js

```diff
@@ -0,0 +1,73 @@
+import {expressThemes, locales, scales, themes} from '../../constants';
+import {makeDecorator} from '@storybook/addons';
+import {Provider, View} from '@adobe/react-spectrum';
+import React, {useEffect} from 'react';
+import './disableAnimations.css';
+
+export const withChromaticProvider = makeDecorator({
+  name: 'withChromaticProvider',
+  parameterName: 'chromaticProvider',
+  wrapper: (getStory, context, {options, parameters}) => {
+    options = {express: false, ...options, ...parameters};
+    let selectedLocales
+    if (options.locales && options.locales.length) {
+      selectedLocales = options.locales;
+    } else {
+      selectedLocales = options.locales ? locales.map(l => l.value).slice(1) : ['en-US'];
+    }
+    // TODO: Decide if more schemes are needed to cover Forced Color issues that could be theme dependent.
```

Comment:
Currently, only running light color scheme and medium scale and english only. Scale and language should be fine but running all color schemes might have benefit to ensure that no contrast changes were introduced.

Reply:
Theoretically, the variables are all being replaced by the HCM variables, so shouldn't make a difference if we're in dark/light/etc. I think the chances of them being actual non-hcm colors is sufficiently unlikely that we don't need to worry about running it across themes. If it's broken in one, it's likely broken in the others, and if it's ok in one, then it's likely ok in the others.

I think we'll get more for our money by focusing on all the states in one theme than not so many states in many themes.